lib/sort_by_length.rb

```diff
@@ -1,7 +1,25 @@
 # A method which will return an array of the words in the string
 #  sorted by the length of the word.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n2)
+# Space complexity: O(1)
 def sort_by_length(my_sentence)
```

Comment:
This works and well done, however the space complexity is O(n) since you're making a new array of all the words.